### ISSUE TITLE
Error getting during deserialization - expected:'"', actual:'o', at offset:5

### ISSUE THREAD

**Issue by vivekspraxa** [neuecc/Utf8Json#181]
I am getting deserialization error , given below , any fix for the below json provided
 

[JsonParsingException: expected:'"', actual:'o', at offset:5]    Utf8Json.JsonReader.ReadStringSegmentRaw() +275    Utf8Json.JsonReader.ReadPropertyNameSegmentRaw() +56    Deserialize(Object[] , JsonReader& , IJsonFormatterResolver ) +298    Utf8Json.Resolvers.Internal.DynamicMethodAnonymousFormatter`1.Deserialize(JsonReader& reader, IJsonFormatterResolver formatterResolver) +205    Utf8Json.JsonSerializer.Deserialize(Byte[] bytes, Int32 offset, IJsonFormatterResolver resolver) +232    Utf8Json.JsonSerializer.Deserialize(Byte[] bytes, IJsonFormatterResolver resolver) +96    Utf8Json.JsonSerializer.Deserialize(String json, IJsonFormatterResolver resolver) +140    Utf8Json.JsonSerializer.Deserialize(String json) +94

And Below is the sample json trying to deserialize this is getting deserialize using newtonsoft

{
  Com: {
    Test: "123"
  }
}

--

**Comment by AbbasKhalili** [comment]
I have the same problem...

**Comment by xygon** [comment]
Does `{
"Com": {
"Test": "123"
}
}` work? Field/property names in JSON should be in quotes.

**Comment by vivekspraxa** [comment]
It work with Newtonsoft but don't working with it

**Comment by LadislavBohm** [comment]
I agree with @xygon that this is not valid JSON. The fact that Newtonsoft JSON.NET works with it is weird to me and I don't think this is an issue of Utf8Json.
